src/test/ui/unsized/issue-71659.rs

```diff
@@ -0,0 +1,94 @@
+#![feature(unsize)]
+
+use std::marker::Unsize;
+use std::rc::Rc;
+use std::sync::Arc;
+
+pub trait CastTo<T: ?Sized>: Unsize<T> {
```

Comment:
Any reason this is not using the minimal version that's in the issue description (as opposed to what's in the playground?)

Reply:
Ah, good point! I assumed that both examples were the same.